app/src/main/java/org/p2p/wallet/common/ui/widget/TransactionSwapImage.kt

```diff
@@ -0,0 +1,52 @@
+package org.p2p.wallet.common.ui.widget
+
+import android.content.Context
+import android.graphics.drawable.PictureDrawable
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.FrameLayout
+import android.widget.ImageView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.RequestBuilder
+import com.bumptech.glide.request.RequestOptions
+import org.p2p.wallet.common.glide.SvgSoftwareLayerSetter
+import org.p2p.wallet.databinding.WidgetTransactionSwapImageBinding
+import org.p2p.wallet.utils.dip
+import org.p2p.wallet.utils.viewbinding.context
+
+private const val IMAGE_SIZE = 29
+
+class TransactionSwapImage @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : FrameLayout(context, attrs, defStyleAttr) {
+
+    private val requestBuilder: RequestBuilder<PictureDrawable> =
```

Comment:
Please, look into the `GlideManager`.
Maybe it's possible to use it here instead of creating the new object